diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index 48e5185..9435d14 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -13,7 +13,7 @@ depends=(atk cairo libxcursor libxinerama libxrandr libxi libepoxy gdk-pixbuf2
          dconf libxcomposite libxdamage pango shared-mime-info at-spi2-atk
          wayland libxkbcommon adwaita-icon-theme json-glib librsvg
          desktop-file-utils mesa cantarell-fonts libcolord rest libcups
-         libcanberra fribidi iso-codes libcloudproviders tracker3
+         libcanberra fribidi iso-codes libcloudproviders
          gtk-update-icon-cache)
 makedepends=(gobject-introspection gtk-doc git glib2-docs sassc meson
              wayland-protocols)
@@ -38,7 +38,7 @@ build() {
   local meson_options=(
     -D broadway_backend=true
     -D cloudproviders=true
-    -D tracker3=true
+    -D tracker3=false
     -D colord=yes
     -D gtk_doc=true
     -D man=true
